src/Arcus.WebApi.Security/Authentication/ConfiguredKey.cs

```diff
@@ -0,0 +1,35 @@
+﻿using GuardNet;
+
+namespace Arcus.WebApi.Security.Authentication
+{
+    /// <summary>
+    /// Represents the configured key on the authentication mechanisms.
+    /// </summary>
+    internal class ConfiguredKey
```

Comment:
Just checking, where is this being used?

Reply:
Ok I've found it but I'm just curious why we actually need it - We guard for null over there so this might be overkill or am I missing something?

Reply:
I added this both for readability (instead of using `string`'s all over and not knowing what it is + securing the use of the configured key to only verify this once and centrilized.

Reply:
🙆‍♂